test/remote/gateways/remote_cyber_source_test.rb

```diff
@@ -129,14 +131,16 @@ def setup
     + '1111111115555555222233101abcdefghijkl7777777777777777777777777promotionCde'
   end
 
+  # Scrubbing is working but will fail at the @credit_card.verification_value assertion
```

Comment:
Hey @yunnydang I think these are probably intermittent failures depending on what ID the Cybersource API returns. I just ran this test a couple of times with the CVV assertion and it passed. I think it's ok to leave your comment above to clarify that the test _may_ intermittently fail if the request ID contains the same substring, but let's not comment out the assertion itself.

We definitely want to know if CVV scrubbing isn't working 😅

Reply:
gotcha! I was going nuts with this since i ran it 6 times and it hit this error every single time LOL. thanks for checking!